tests/queries/0_stateless/03321_create_table_as_replicated_engine_args.sql

```diff
@@ -0,0 +1,17 @@
+-- Tags: zookeeper, no-replicated-database, no-parallel, no-ordinary-database
+-- no-replicated-database: we explicitly run this test by creating a replicated database test_03321
+
+DROP DATABASE IF EXISTS test_03321;
+CREATE DATABASE test_03321 ENGINE=Replicated('/clickhouse/databases/test_03321', 'shard1', 'replica1');
```

Comment:
Avoid `no-parallel` whenever possible:
```suggestion
-- Tags: zookeeper, no-replicated-database, no-parallel, no-ordinary-database
-- no-replicated-database: we explicitly run this test by creating a replicated database test_03321

DROP DATABASE IF EXISTS test_03321;
CREATE DATABASE {CLICKHOUSE_DATABASE:Identifier} ENGINE=Replicated('/clickhouse/databases/test_03321/{database}', 'shard1', 'replica1');
```
(another option would be rewriting it in bash)